testpilot-metrics.js

```diff
@@ -24,15 +24,15 @@
  */
 function Metrics({id, version, uid, tid = null, type = 'webextension', debug = false}) {
   if (!id) {
-    throw new SyntaxError(`'id' property is required.`);
+    throw new SyntaxError("'id' property is required.");
```

Comment:
In this case, I think ESLint prefers you only use the template backticks as a delimiter if you're interpolating variables.
Actually, when auto-running `eslint . --fix`, it converted those strings to use single quotes like: `throw new SyntaxError('\'id\' property is required');` (but I find the escaping of characters a bit harder to read, so I converted to double quotes). We can try alternate solutions if you want.